util/topgen/templates/toplevel_pkg.sv.tpl

```diff
@@ -5,24 +5,20 @@ ${gencmd}
 <%
 import topgen.lib as lib
 
-
 def camelcase(str):
     """Turns a string from 'snake_case' to 'CamelCase'."""
-    return "".join([part.capitalize() for part in str.split("_")])
-
+    return lib.Name.from_snake_case(str).as_camel_case()
 
-def get_dio_pin_enum_literal(sig, start_idx):
+def get_dio_pin_enum_literal(sig):
     """Returns the DIO pin enum literal with value assignment"""
-    if "width" in sig and sig["width"] > 1:
-        return_str = ""
-        for i in range(int(sig["width"])):
-            if i > 0:
-              return_str += "\n    "
-            return_str += camelcase("top_{}_dio_pin_{}_{} = {},".format(
-            top["name"], sig["name"], i, start_idx + i))
-        return return_str
-    return camelcase("top_{}_dio_pin_{} = {},".format(
-        top["name"], sig["name"], start_idx))
+    if sig['idx'] < 0:
+      return_str = camelcase("top_{}_dio_pin_{} = {},".format(
+        top["name"], sig["name"], sig['glob_idx']))
+    else:
+      return_str += camelcase("top_{}_dio_pin_{}_{} = {},".format(
+        top["name"], sig["name"], sig['glob_idx'], sig['idx']))
```

Comment:
Can you use `Name.get_io_enum_literal()` instead?

Reply:
Yes it looks like this is a stale function, so it can be deleted entirely.